continuous-delivery/pull_pypirc.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import boto3
+import base64
+import os
+import argparse
+from botocore.exceptions import ClientError
+
+
+def get_secret(stage):
+
+    secret_name = '{}/aws-crt-python/.pypirc'.format(stage)
```

Comment:
Why not just pull this with the aws cli like the rest of the secrets?

Reply:
good point.